openpgp/ecdh/x25519.go

```diff
@@ -31,14 +31,14 @@ func x25519GenerateKeyPairBytes(rand io.Reader) (priv [32]byte, pub [32]byte, er
 	// 2^{254} + 8 * [0, 2^{251}), in order to avoid the small subgroup of
 	// of the curve.
 	//
-	// This masking is done internally to ScalarBaseMult and so is unnecessary
+	// This masking is done internally to KeyGen and so is unnecessary
 	// for security, but OpenPGP implementations require that private keys be
 	// pre-masked.
 	priv[0] &= 248
 	priv[31] &= 127
 	priv[31] |= 64
```

Comment:
AFAIK `x25519.KeyGen` takes a pointer to the private and takes care of the clamping too